worker/task_executor.go

```diff
@@ -0,0 +1,164 @@
+package worker
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/docker/client"
+	"github.com/ufcg-lsd/arrebol-pb-worker/utils"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+)
+
+const (
+	TaskScriptExecutorFileName  = "task-script-executor.sh"
```

Comment:
should we use this hardcoded path (TaskScriptExecutorFileName)?

Reply:
I didn't understand...

Reply:
should we have this on code or read from a config file?

Reply:
I guess that there is no problem with this harcoded path, since it is a "static" file placed in a very specific path.

Reply:
Ok